templates/standard.adoc

```diff
@@ -1,48 +1,109 @@
-= Pomegranates
+= `Vehicle`
 include::../include/config.adoc[]
-:description: A sample template for using AsciiDoc.
-:keywords: sample
+:description: The standard style template for documentation.
+:keywords: standard, template
 
-This is a sample template of how a good AsciiDoc can be structured.
+This is the standard template of how a good AsciiDoc should be structured.
 This pragraph describes the purpose of the document.
-In this sample, an imaginary API
-for producing pomegranates will be documented.
+In this example, an imaginary API for a vehicle will be documented.
 
-So without further ado, here is what we should see when we are done.
+== `Vehicle` Properties
 
-[#img-pomegranate]
-.The Expected Result
-image::pomegranate.png[Pomegranate]
+=== `make: string`
+The make of the vehicle.
 
-== Pomegranate API Methods
+=== `model: string`
+The model of the vehicle.
 
-=== `plant_tree(pos, name)`
-* `pos`: `vector`
-* `name`: `string`
+=== `year: number`
+The year of the vehicle.
 
-Plants a tree of type `name` at `pos`
-if `name` is a registered pomegranate tree.
+=== `trim: string`
+The trim of the vehicle.
 
-.Planting A Tree
+=== `color: ColorSpec`
+The color of the vehicle.
```

Comment:
Stating the obvious?

Reply:
Would you like it to be empty? Perhaps a header list is not the best option for properties.

Reply:
I'd be fine with it being empty.